src/main/java/dekvall/pushnotification/PushNotificationsPlugin.java

```diff
@@ -22,7 +22,7 @@
 
 @Slf4j
 @PluginDescriptor(
-	name = "Push Notifications"
+		name = "Push Notifications"
```

Comment:
And dedent this final one as well please :)

Reply:
Wow, I really don't have a good eye for spotting these things :)